testData/invalid/json/5_invalid_intrinsic_select_1.json

```diff
@@ -0,0 +1,245 @@
+{
```

Comment:
Due to the size of these templates, it not particularly clear where the error specified in the test should be occurring. These would be nicer as smaller templates where the test case to fail is easy to spot, and relatively isolated. I.e. no parameters and only features which are required for the test to fail in the specified way.

Reply:
Yeah, I think I'll try to move my tests into something more a like a unit test. That will make it much more clear whats happening.

Reply:
These look much better!